src/pages/code-reviews-getting-started/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Code reviews: getting started"
+date: 2018-05-03T12:07:50+03:00
+draft: false
+---
+
+What if I told you there is this really straightforward technique to make your project run smoother, faster, and with fewer bugs 🤔. Better yet, it’s completely free, requires only minimal tooling, and any developer can learn to use it in a matter of minutes 😳. This is how good code reviews are, and why you should use them.
+
+_Author: Thibaud Colas, Software Architect at Codemate Ltd._
+
+---
+
+## Why it’s good
+
+Developers tend to think of productivity in terms of “how fast can I write the code for this new feature”. We also tend to say “I write code” when describing our jobs. In practice though, professional software development is a matter of understanding the code that is already there: we simply spend much more time reading than writing. Productivity thus depends more on the overall code’s quality and maintainability, rather than the skills of individual contributors making changes.
+
+Joel Spolsky puts it neatly:
+
+> [It’s harder to read code than to write it](https://www.joelonsoftware.com/2000/04/06/things-you-should-never-do-part-i/).
+
+John Papa recently gave a great talk about what code readability means in practice:
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/56mETnrByBM" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```

Comment:
I would like to have a short recap about the main points of the video here.

Some key points to consider, don't have to use all of them:

* speaker is John Papa, developer advocate working for Microsoft
* you read code more than you write it, according to Robert C. Martin the read/write ratio is "well over 10:1"
* what makes it worse, like Joel Spolsky said earlier, reading code is harder than writing
* favor fast and effective communication over clever and fast code
* clean, readable code helps to avoid rewrites
* use consistent and meaningful names in code
* self-documenting code is better than commenting everything
* write dirty code that works; then clean it

Reply:
I think the video would be a good watch in its own right, and I don't want to break the flow of the article with too many recaps, so I'll leave this as-is.